packages/workshop-utils/src/epic-api.server.ts

```diff
@@ -138,13 +153,16 @@ async function getEpicVideoInfo({
 					? getEpicAIVideoAPIUrl(epicVideoEmbed)
 					: `https://${epicUrl.host}/api${epicUrl.pathname}`
 
+			log(`making API request to: ${apiUrl}`)
 			const infoResponse = await fetch(
 				apiUrl,
 				accessToken
 					? { headers: { authorization: `Bearer ${accessToken}` } }
 					: undefined,
 			)
 			const { status, statusText } = infoResponse
+			log(`API response: ${status} ${statusText}`)
+			
```

Comment:
I would like more logging below. When there's an error I need to have enough information to debug the error.